Elevel/Application/Features/ReportCommands/CreateReportCommand.cs

```diff
@@ -31,6 +33,8 @@ public Validator()
             {
                 this.CascadeMode = CascadeMode.Stop;
                 
+                RuleFor(x => x).NotEmpty();
```

Comment:
request can't be null if you will not set it to null and call mediatr, but that has no sense

Reply:
Deleted it